index.ts

```diff
@@ -79,12 +79,13 @@ export function missingTranslationHandler() {
 })
 export class NgJhipsterModule {
     static forRoot(providedConfig: ModuleConfig): ModuleWithProviders {
-        ConfigHelper.setModuleConfigOptions(providedConfig);
```

Comment:
We can't have fonction calls if using AOT compilation else it will throw this error  : 

`Error encountered resolving symbol values statically. Function calls are not supported. Consider replacing the function or lambda with a reference to an exported function`